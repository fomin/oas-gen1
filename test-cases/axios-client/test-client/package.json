{
  "name": "axios-client",
  "version": "1.0.0",
  "description": "",
  "main": "out/index.js",
  "types": "out/index.d.ts",
  "scripts": {
    "build": "tsc",
    "start": "tsc && node out/index.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "tslib": "^1.10.0",
    "expected": "file:../expected",
    "@js-joda/core": "^2.0.0",
    "axios": "^0.21.1"
  }
}
