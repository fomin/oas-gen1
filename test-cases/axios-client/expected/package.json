{
  "name": "axios-client",
  "version": "1.0.0",
  "description": "",
  "main": "out/simple.js",
  "types": "out/simple.d.ts",
  "scripts": {
    "build": "tsc",
    "start": "tsc && node out/simple.js",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "axios": "^0.21.1",
    "tslib": "^1.10.0",
    "@js-joda/core": "^2.0.0",
    "oas-gen-support": "file:../../../npm-libs/oas-gen-support"
  }
}
